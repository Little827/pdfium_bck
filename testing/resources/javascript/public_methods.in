{{header}}
{{object 1 0}} <<
  /Type /Catalog
  /Pages 2 0 R
  /AcroForm << /Fields [ 4 0 R 10 0 R ] /DR 5 0 R >>
>>
endobj
{{object 2 0}} <<
  /Count 1
  /Kids [ 3 0 R ]
  /Type /Pages
>>
endobj
{{object 3 0}} <<
  /Type /Page
  /Parent 2 0 R
  /Resources 5 0 R
  /MediaBox [ 0 0 300 200 ]
  /Contents 8 0 R
  /Annots [ 4 0 R 10 0 R ]
>>
endobj
{{object 4 0}} <<
  /Type /Annot
  /FT /Tx
  /T (Text Box)
  /DA (0 0 0 rg /F1 12 Tf)
  /Rect [ 100 100 200 130 ]
  /Subtype /Widget
  /AA <<
    /F 9 0 R
  >>
>>
endobj
{{object 5 0}} <<
  /Font 6 0 R
>>
endobj
{{object 6 0}} <<
  /F1 7 0 R
>>
endobj
{{object 7 0}} <<
  /Type /Font
  /Subtype /Type1
  /BaseFont /Helvetica
>>
endobj
{{object 8 0}} <<
>>
stream
endstream
endobj
{{object 9 0}} <<
  /JS (
    app.alert('*** starting test ***');
    try {
      AFDate_Format(1);
      AFDate_FormatEx('blooey');
      AFDate_Keystroke(2);
      AFDate_KeystrokeEx('blooey');
      AFExtractNums('100x faster');
      AFMakeNumber('blooey');
      AFMergeChange('blooey');
      AFNumber_Format(0, 1, 0, 0, "", false);
      AFNumber_Keystroke(1, 2);
      AFParseDateEx(1, 2);
      AFPercent_Format(1, 2);
      AFPercent_Keystroke(1, 2);
      AFRange_Validate(1, 2, 3, 4);
      AFSimple(1, 2, 3);
      AFSimple_Calculate(1, 'foo');
      AFSpecial_Format(3);
      AFSpecial_Keystroke(65);
      AFSpecial_KeystrokeEx('999999');
      AFTime_Format(1);
      AFTime_FormatEx('blooey');
      AFTime_Keystroke(65);
      AFTime_KeystrokeEx('blooey');
    } catch (e) {
      app.alert("ERROR: " + e);
    }
    app.alert('*** ending test ***');
  )
  /S /JavaScript
>>
endobj
{{object 10 0}} <<
  /Type /Annot
  /FT /Tx
  /T (Text2)
  /DA (0 0 0 rg /F1 12 Tf)
  /Rect [ 100 40 200 70 ]
  /Subtype /Widget
>>
endobj
{{xref}}
{{trailer}}
{{startxref}}
%%EOF
