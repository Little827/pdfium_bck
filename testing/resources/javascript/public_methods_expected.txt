Alert: *** starting test ***
[icon=3,type=0]: The input value is invalid.
[icon=3,type=0]: The input value can't be parsed as a valid date/time (H).
Alert: *** ending test ***
